### ISSUE TITLE
About accessibility rule

### ISSUE THREAD

**Issue by mrm7moud** [FACG2/Horizons#9]
you have 2 error and 5 warning 

[Severe] An element's ID must be unique in the DOM (3)
[Severe] Controls and media elements should have labels (4)
[Warning] Images should have a text alternative or presentational role (17)
[Warning] Text elements should have a reasonable contrast ratio (10)
[Warning] The purpose of each link should be clear from the link text (1)
[Warning] The web page should have the content's human language indicated in the markup (1)
[Warning] These elements are focusable but either invisible or obscured by another element (5)

**Comment by Eslam3bed** [comment]
Thanks Alot mrm7moud ,
